### DISCUSSION TITLE
Word mismatch in the book?

### DISCUSSION THREAD

**Discussion by hindermath** [holgerflick/flixaccounting#29] [Errata]
In Chapter 4.3 (page 128), 2nd paragraph, last sentence is written `Create directory`. But the button in Figure 4.2 (on page 129) shows the caption `Create dictionary` and in the rest of the book also refers to _dictionary_. So, I think, that was an undetected typo in the book? Or I'm wrong and missed something?

**Comment by holgerflick** [comment]
Good catch. I replaced it before publication in another location. Its corrected in my source code. As soon as I push another version to Amazon it will be fixed. 

Thank you.

**Reply by hindermath** [reply]
Of course, absolutely. I'll work through the book one by one and try to understand everything. That's when I noticed this discrepancy. It's a good thing that you can change and correct this in the next batch of books.
